src/absil/il.fs

```diff
@@ -1358,31 +1358,38 @@ type ILGenericParameterDef =
 
 type ILGenericParameterDefs = ILGenericParameterDef list
 
+let memberAccessOfFlags flags =
+    let f = (flags &&& 0x00000007)
+    if f = 0x00000001 then  ILMemberAccess.Private 
+    elif f = 0x00000006 then  ILMemberAccess.Public 
+    elif f = 0x00000004 then  ILMemberAccess.Family 
+    elif f = 0x00000002 then  ILMemberAccess.FamilyAndAssembly 
+    elif f = 0x00000005 then  ILMemberAccess.FamilyOrAssembly 
+    elif f = 0x00000003 then  ILMemberAccess.Assembly 
+    else failwith "impossible"
+
+let convertMemberAccess (ilMemberAccess:ILMemberAccess) =
+    match ilMemberAccess with
```

Comment:
ok this is more a speculation: shouldn't we order the cases by probabilty of being called. Like I assume "Public" would be very frequent and if we move it up, then we reduce comparisions!?

Reply:
In theory but only if there is evidence it's important to perf